### PR TITLE
{Keyvault} Fix keyvault packaging tests

### PR DESCRIPTION
**Pull request by evelyn-ys**
**Description**

Our CI fails in ([CI link](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1001522&view=logs&jobId=1e6fba43-bdcf-51f0-a062-3712c33fbb51&j=1e6fba43-bdcf-51f0-a062-3712c33fbb51&t=a1e787eb-ec67-5c3d-79ec-0c68f4f59d65))
image
with such error
```
Can't overwrite existing cassette ('/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/keyvault/tests/latest/recordings/test_keyvault_certificate_crud.yaml') in your current record mode ('once').
No match for the request (<Request (GET) https://cli-test-kv-ct-000002.vault.azure.net/secrets/cert2/?api-version=7.0>) was found.
```
when runs into L1895
https://github.com/Azure/azure-cli/blob/562eeaa45ddce309d3ff30723595ce3820631af5/src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py#L1892-L1895

L1895 is trying to download secret if `cert_secret_path` not found locally.
It fails now in testing yum/deb package because:
- Our packaging script doesn't include file without file extension. As a result, the `cert_secret_path` (which is `TEST_DIR/cert_secret`) is never packaged so `not os.path.exists(cert_secret_path)`(L1892) returns true
- The latest `test_keyvault_certificate_crud.yaml`(recorded by #18520) doesn't contain downloading secret request

The previous `test_keyvault_certificate_crud.yaml` was recorded without `TEST_DIR/cert_secret` locally. So the yaml file contained downloading request, and the `TEST_DIR/cert_secret` was saved. So for local test, it could skip downloading(L1892 `if not os.path.exists(cert_secret_path)`) and use this file directly; for yum/deb package test, it could succeed finding matched downloading request in yaml. All was well.

But #18520 re-record `test_keyvault_certificate_crud.yaml`. Since there already exists `TEST_DIR/cert_secret` locally, the newly created yaml doesn't contain downloading secret request and it works well in local test but crashes in yum/deb package test.

This PR re-records yaml file with downloading secret request and deletes `cert_secret` permanently.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Keyvault

File: src/azure-cli/azure/cli/command_modules/security/_help.py [hunk 1]
```diff
@@ -90,6 +90,11 @@
         az security atp storage update --resource-group MyResourceGroup --storage-account MyStorageAccount --is-enabled false
 """
 
+helps['security va'] = """
+type: group
+short-summary: View Vulnerability Assessment.
+"""
+
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/security/_help.py:97]: Please split it to a separate PR or merge from `dev` to incorporate the change.

**Review by jsntcy — APPROVED** [review]

**Comment by jiasli** [comment]
Please make a short statement about why the test didn't fail before.

File: src/azure-cli/azure/cli/command_modules/security/_help.py (hunk 1 above)
Comment (evelyn-ys) [reply src/azure-cli/azure/cli/command_modules/security/_help.py:97]: If I split to other PR, it will fail in keyvault test package.
For now, `dev` doesn't cover this help fix.

This is already a split PR fixed for #18868. Just want to make things simple.

Comment (evelyn-ys) [reply src/azure-cli/azure/cli/command_modules/security/_help.py]: merged with `dev`

**Comment by evelyn-ys** [comment]
> Please make a short statement about why the test didn't fail before.

PR description updated

**Review by yonzhan — APPROVED** [review]
Nice PR description. A very great example for drilling down to the bottom:)

**Review by jiasli — APPROVED** [review]
